inginious/frontend/courses.py

```diff
@@ -170,6 +171,10 @@ def get_access_control_method(self):
         """ Returns either None, "username", "binding", or "email", depending on the method used to verify that users can register to the course """
         return self._registration_ac
 
+    def get_access_control_type(self):
+        """ Returns either "accept" or "deny", depending on the control type used to verify that users can register to the course """
+        return self._registration_ac_type
```

Comment:
As there are only 2 possible states (allow or deny), I'd rather have a boolean data type instead of a textual one. This would lower the risk of bugs and ease code reading.

Reply:
Fixed in 49ba9d4f6a511a7785f696823835e68da5184a92.